### PR TITLE
minor change

### PR DESCRIPTION
**Pull request by trinhtan**

### CODE REVIEW

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Comment by DuongNT1512** [comment]
conflict

**Comment by DuongNT1512** [comment]
conflicted

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

File: config/locales/en.yml [hunk 1]
```diff
@@ -52,4 +52,4 @@ en:
       reckon_link: "X e m"
       edit_link: "Sửa"
       delete_checkbox: "Xóa"
-      delete_button: "Xóa"
+
```
Comment (thanhhungchu95) [inline config/locales/en.yml]: xóa space thừa đi nhé

**Comment by thanhhungchu95** [comment]
commented

**Comment by trinhtan** [comment]
https://github.com/framgia-edu/exam_scheduling_01/pull/15

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Pull request merged**